src/test/components/elements/inputs/RegistrationSubmit.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { screen } from "@testing-library/react";
+import { RegistrationSubmit } from "../../../../app/components/elements/inputs/RegistrationSubmit";
+import { renderTestEnvironment } from "../../../utils";
+
+describe("RegistrationSubmit", () => {
+  const setupTest = (props = {}) => {
+    renderTestEnvironment({
+      role: "ANONYMOUS",
+      PageComponent: RegistrationSubmit,
+      componentProps: {
+        isRecaptchaTicked: true,
+        ...props,
+      },
+      initialRouteEntries: ["/register/student"],
+    });
+  };
+
+  it("renders the component, and submit button is enabled when isRecaptchaTicked is true", async () => {
```

Comment:
You're testing two completely different things here, so they should be two tests (one to check that the component renders, the other one to check that when isRecaptchaTicked is true then the submit button is enabled).

Reply:
Separated into 2 tests